app/Http/Controllers/PasswordSecurityController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use PragmaRX\Google2FA\Google2FA;
+use Illuminate\Support\Facades\Auth;
+use App\Models\PasswordSecurity;
+
+class PasswordSecurityController extends Controller
+{
+    public function show2faForm()
```

Comment:
Would be nice to have at least a comment before each function describing what it does in a sentence